### PR TITLE
1 loginf

### PR DESCRIPTION
**Pull request by JoniPahikainen**

### CODE REVIEW

File: frontend/src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import "../css/LR.scss";
+import React, { useState } from "react";
+import Axios from "axios";
```
Comment (tomlnd) [inline frontend/src/components/Login.js:3]: Import axios from ../axios.js instead

Comment (tomlnd) [inline frontend/src/components/Login.js:3]:
```suggestion
import axios from "../axios";
```

File: frontend/src/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    method: "post",
+    url: url,
+    data: body,
+    headers: {
+      "Content-Type": "application/json",
+    },
+    json: true,
+  };
+  const login = () => {
+    Axios(authOptions)
```
Comment (tomlnd) [inline frontend/src/components/Login.js:27]:
```suggestion
    axios(authOptions)
```

**Review by Kesonjus — COMMENTED** [review]
axios is good

**Pull request merged**